src/api-reference/invoice/v3.purchase-order.markdown

```diff
@@ -78,42 +78,50 @@ Create or update a Purchase Order. Batch processing is not available using the P
   "CurrencyCode": "USD",
   "OrderDate":"2011-08-12T20:17:46.384Z",
   "ID": "API101",
-  "IsTest": "N",
+  "IsTest": "false",
```

Comment:
Is this a String that's using boolean logic?
If not, the boolean value should not be within quotation marks.

Reply:
Fixed in 51a69ac